apis/Google.Cloud.Spanner.Data/docs/connection_string.md

```diff
@@ -175,4 +175,14 @@ The valid type mappings for `SpannerDbType.Date` are:
 Examples:
 
 - SpannerToClrTypeDefaultMappings=DateToDateTime
-- SpannerToClrTypeDefaultMappings=DateToSpannerDate
\ No newline at end of file
+- SpannerToClrTypeDefaultMappings=DateToSpannerDate
+
+## DatabaseRole
+
+The database role used while creating sessions for a database. This is only used when fine grained
+access control (FGAC) is enabled for the spanner instance otherwise this is ignored. In-case when FGAC is enabled
+and database role is not provided explicitly or may be null in that case default (public) role will be used.
```

Comment:
(And remove all other references to FGAC if you have any, the acronym does not appear on public docs.)

```suggestion
The database role associated to this connection string. This option is only used if Fine Grained
Access Control is enabled for the Spanner instance. If Fine Grained Access Control is enabled for this Spanner instance, and a database role is not provided the default (public) role will be used.
```